src/libs/actions/Device/generateDeviceID/index.android.js

```diff
@@ -0,0 +1,32 @@
+import DeviceInfo from 'react-native-device-info';
+import Str from 'expensify-common/lib/str';
+
+const deviceID = DeviceInfo.getDeviceId();
+const uniqueID = Str.guid(deviceID);
+
+/**
+ * Get the "unique ID of the device". Note that the hardware ID provided by react-native-device-info for Android is considered private information,
+ * so using it without appropriate permissions could cause our app to be unlisted from the Google Play Store:
```

Comment:
Is this not also true for iOS and other platforms?

Reply:
According to my research, it is not true for iOS or macOS desktop

Reply:
i.e: on iOS we can use the true device ID